### PR TITLE
More random decomps

### PR DESCRIPTION
**Pull request by Solitai7e**

### CODE REVIEW

File: src/code_0800A868.c [hunk 1]
```diff
@@ -31,3 +31,23 @@ s32 sub_0800A868(u8 playerId, struct Object *obj)
 	
     return var2;
 }
+
+void sub_0800A8BC(void)
+{
+    gUnk_03000004 = 0;
```
Comment (jiangzhengwenjz) [inline src/code_0800A868.c:37]: Does `*(u32 *)(IWRAM + 4) = 0` match? We've seen the pattern elsewhere so the symbol probably doesn't exist

File: src/code_0800A91C.c [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#include "global.h"
+#include "functions.h"
+
+// TODO: define file boundaries
```
Comment (jiangzhengwenjz) [inline src/code_0800A91C.c:4]: Is there a reason for you to put the function in a separate file, rather than keeping it in `code_0800A868.c`?

Comment (Solitai7e) [reply src/code_0800A91C.c:4]: That's what I was going to do originally but then it turns out there's already an existing declaration in `functions.h`, and so after consulting @Cleverking2003 I was told to move it to it's own file. (and leave the declaration in `functions.h`)

Comment (Cleverking2003) [reply src/code_0800A91C.c:4]: It seems like I confused you a bit. I should have said you should have left definition in the file where it was before and not split it.

File: src/code_0800A868.c [hunk 3]
```diff
@@ -1,6 +1,8 @@
 #include "code_0800A868.h"
 #include "functions.h"
 
+#include "gba/defines.h"
```
Comment (jiangzhengwenjz) [inline src/code_0800A868.c:4]: I think it's already included in other header?

Comment (Solitai7e) [reply src/code_0800A868.c]: Didn't realize it was `IWRAM_START` instead of `IWRAM` at first, my bad.

Comment (jiangzhengwenjz) [reply src/code_0800A868.c]: :man_facepalming: my bad

**Comment by jiangzhengwenjz** [comment]
@Solitai7e Note that I've squashed your commits, so you can do something like `git fetch upstream && git reset --hard upstream/master` to sync with upstream

**Pull request merged**